README.md

```diff
@@ -1,2 +1,65 @@
-# Hello_m-croverse
-first project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+hello microvers
+
+> Description the project.
+> this is my first project in microvers
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
```

Comment:
Kindly include the technologies you have used for this project. This will make your README.md file look professional.